internal/tracker/tracker.go

```diff
@@ -182,15 +223,30 @@ func Track(ctx context.Context, urls []string, input []byte, prune bool, freshen
 		}
 	}
 
-	t.filterBundles(prune)
+	return nil
+}
 
-	t.setExpiration()
+func trackGitReferences(_ context.Context, urls []string, t *Tracker) error {
+	effective_on := effectiveOn()
 
-	if err := t.convertToOldFormat(); err != nil {
-		return nil, err
+	for _, u := range urls {
+		repository, rest, found := strings.Cut(u, "//")
```

Comment:
Does this mean we have to specify `git+ssh` or `git+https` for task references (used by git resolver) when calling the `ec track`  command?

Reply:
https://github.com/enterprise-contract/ec-policies/blob/main/example/data/trusted_tekton_tasks.yml#L3

Reply:
Interesting, when handling on the policy side we might ignore whether the scheme is `git+ssh` or `git+https` or `git+...`, we might not care at all about the transfer protocol...